packages/test-helpers/src/withComponent.js

```diff
@@ -18,9 +25,36 @@ export default () => {
       client.disconnect();
     };
 
-    const fructose = {};
-    fructose.loadComponent = loadComponent;
-    fructose.disconnect = disconnect;
+    const snapshotTest = async (platform, testname) => {
+      const snapsPath = `${testDir}/__snapshots__`;
+      const snaps = new AppSnaps(platform, snapsPath);
+
+      if (snaps.exists(testname)) {
+        console.warn("IT EXISTS");
```

Comment:
Might be worth logging out what exists, I think it would be useful once running CI